doc/string/chomp.rdoc

```diff
@@ -0,0 +1,29 @@
+Returns a new string copied from +self+, with trailing characters possibly removed:
+
+When +line_sep+ is <tt>"\n"</tt>, removes the last one or two characters
+if they are <tt>"\r"</tt>, <tt>"\n"</tt>, or <tt>"\r\n"</tt>
+(but not <tt>"\n\r"</tt>):
+
+  $/                    # => "\n"
+  "abc\r".chomp         # => "abc"
+  "abc\n".chomp         # => "abc"
+  "abc\r\n".chomp       # => "abc"
+  "abc\n\r".chomp       # => "abc\n"
+  "тест\r\n".chomp      # => "тест"
+  "こんにちは\r\n".chomp  # => "こんにちは"
+
+When +line_sep+ is <tt>''</tt> (an empty string),
+removes multiple trailing occurrences of <tt>"\n"</tt> or <tt>"\r\n"</tt>
+(but not <tt>"\r"</tt> or <tt>"\n\r"</tt>):
+
+  "abc\n\n\n".chomp('')           # => "abc"
+  "abc\r\n\r\n\r\n".chomp('')     # => "abc"
+  "abc\n\n\r\n\r\n\n\n".chomp('') # => "abc"
+  "abc\n\r\n\r\n\r".chomp('')     # => "abc\n\r\n\r\n\r"
+  "abc\r\r\r".chomp('')           # => "abc\r\r\r"
+
+When +line_sep+ is neither <tt>"\n"</tt> nor <tt>''</tt>,
+removes a single trailing line separator if there is one:
+
+  "abcd\n".chomp   # => "abcd"
+  "abcd\n\n".chomp # => "abcd\n"
```

Comment:
I'm not sure this is true, for example, `"o"` is none of the two mentioned:

```ruby
irb(main):020:0> "helloo\n".chomp("o")
=> "helloo\n"
```

And instead, it just removes the `"o"` at the end:

```ruby
irb(main):021:0> "helloo".chomp("o")
=> "hello"
```

Reply:
Duh.  The text is correct, but the examples are wrong; they don't even fulfill the hypothesis in the text.  Fixed.